fachhochschule-vorarlberg-note.csl

```diff
@@ -383,7 +383,7 @@
     </choose>
   </macro>
   <citation et-al-min="4" et-al-use-first="1" et-al-subsequent-min="4" et-al-subsequent-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year">
-    <layout prefix="(" suffix=")" delimiter="; ">
+    <layout delimiter="; ">
```

Comment:
@POBrien333, could you check if this change makes sense to you? As a note style, I guess parentheses aren't used, but I didn't check the style guide.

Reply:
Hi, in the FHV Style Guide on page 17 are examples, where you can see that there are no parentheses. I have marked them.

[FHV_Style_Guide.pdf](https://github.com/citation-style-language/styles/files/1817332/FHV_Style_Guide.pdf)